packages/neuvector/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,32 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,10 +1,28 @@
++annotations:
++  catalog.cattle.io/auto-install: neuvector-crd=match
++  catalog.cattle.io/certified: rancher
++  catalog.cattle.io/display-name: NeuVector
++  catalog.cattle.io/kube-version: '>=1.18.0-0'
++  catalog.cattle.io/namespace: cattle-neuvector-system
++  catalog.cattle.io/os: linux
++  catalog.cattle.io/permit-os: linux
++  catalog.cattle.io/provides-gvr: neuvector.com/v1
++  catalog.cattle.io/rancher-version: '>= 2.6.5-0'
```

Comment:
this should be `catalog.cattle.io/rancher-version: '>= 2.6.5-0 <= 2.6.100-0'`

Reply:
we will update it.